azure/datalake/store/utils.py

```diff
@@ -110,10 +109,14 @@ def seek_delimiter(file, delimiter, blocksize):
     blocksize: int
         Number of bytes to read from the file at once.
     """
-
+    # if we are at either the very beginning or very end of the file, 
+    # there is nothing to return so just return
```

Comment:
Based on the logic as it is described here, I believe this to be true. The problem is that I am not entirely confident in any of the delimiter logic, however it is not currently hooked up, and I will spend more time validating this in the future.

The comment here is to explain the existing logic, as I did not change the existing logic. Comments are very helpful for maintainability :)